lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/BinarySpec.hs

```diff
@@ -76,87 +80,109 @@ import System.Directory
-                        , SlotsPerEpoch $ W.EpochLength 2160
-                        , SlotDuration 15
-                        , EpochStabilityDepth (Quantity 10)
-                        , AddBftLeader $ LeaderId $ unsafeFromHex
-                            "30a694b80dbba2d1b8a4b55652b03d96\
-                            \315c8414b054fa737445ac2d2a865c76"
-                        , ConsensusGenesisPraosParamF (Milli 220)
+                        , Consensus GenesisPraos
+                        , ConfigLinearFee $ LinearFee (Quantity 42) (Quantity 0)
+                        , SlotsPerEpoch (W.EpochLength 3)
+                        , SlotDuration 1
+                        , KesUpdateSpeed (Quantity 43200)
+                        , ConsensusGenesisPraosParamF (Milli 1000)
+                        , BftSlotsRatio (Milli 0)
                         , MaxNumberOfTransactionsPerBlock 255
-                        , BftSlotsRatio (Milli 220)
-                        , AllowAccountCreation True
-                        , ConfigLinearFee $ LinearFee (Quantity 0) (Quantity 0)
+                        , EpochStabilityDepth (Quantity 4)
+                        , AddBftLeader $ LeaderId $ unsafeFromHex
+                            "6555f34e4de7ef3b01e889dba07a399b\
+                            \3a3ea0c36f21184d01439af96c4442eb"
                         ]
                     , Transaction (Tx
                         { txid = Hash $ unsafeFromHex
-                            "6f5e01c34590f5ead789c234a816ac25\
-                            \41baece53f6e51bf52222bd7feb5cd04"
+                            "30b99c425ca5aa64e24f23b5cef542\
+                            \170ac96ea32ea823904f9446cd49966013"
                         , inputs = []
                         , outputs =
                             [ TxOut
-                                { address = unsafeDecodeAddress @'Testnet
-                                    "ta1svejfsmcd8qjy6y6xkghmaf6fu3f\
-                                    \fga99a59up04lrjnhpl8afzj7w4yyxw"
-                                , coin = Coin 14
+                                { address = Address $ unsafeFromHex
+                                    "85877fbb2283ba1ed56d835fb8cd66694a8\
+                                    \40360e69c690a04aeef39629cdd804f"
+                                , coin = Coin 1
                                 }
                             ]
                         }, [])
-                    , UnimplementedMessage 1
-                    , UnimplementedMessage 3
+                    , UnimplementedMessage 5
+                    , TransactionWithDelegation (
+                            PoolId (unsafeFromHex
+                                    "a09ae0da1e618eeb09e7b78d73e265af18f8\
+                                    \7d4d5320386ebf0235f54ecd0347")
+                            , ChimericAccount (unsafeFromHex
+                                    "877fbb2283ba1ed56d835fb8cd66694a840\
```

Comment:
And related to https://github.com/input-output-hk/cardano-wallet/pull/932/files#r341079348

This does indeed seem to be the public key of the account address in the genesis.yaml, which also makes sense
```bash
$ jcli address info ca1skrhlwezswapa4tdsd0m3ntxd99ggqmqu6wxjzsy4mhnjc5umkqy7pccwfa
discrimination: testing
account: ed25519_pk1salmkg5rhg0d2mvrt7uv6enff2zqxc8xn35s5p9wauuk98xasp8sdkarvz

$ echo "ed25519_pk1salmkg5rhg0d2mvrt7uv6enff2zqxc8xn35s5p9wauuk98xasp8sdkarvz" | jcli key to-bytes
877fbb2283ba1ed56d835fb8cd66694a840360e69c690a04aeef39629cdd804f
```